### DISCUSSION TITLE
WIll the MacOS app get any of the updates?

### DISCUSSION THREAD

**Discussion by npaine01** [chidiwilliams/buzz#1014] [General]
About a year ago, the apple app had a slightly more polished interface, and I was happy to pay and contribute to the great work that is going on to create this project.

I didn't realize that it wouldn't get any updates, but now I see the open source version has had a few since the last Mac OS version (currently 1.0.2, last updated 9 months ago?). 

Should we just be using the open source one?

**Comment by raivisdejus** [comment]
I have no information on the App store version development plans. It is partly separate from open source version, most likely shares many common parts, but separate and not open source.

In the end I think we have an option to use the version that suits our needs. If App store version does what you need, you can enjoy the more polished UI, if you need some feature that is available only in the open source version for now, you can use it.

The speech recognition power comes from the models used, so that should be the same in both versions.
